app.py

```diff
@@ -1,8 +1,22 @@
 from flask import Flask, render_template
+from flask_wtf import FlaskForm
+from flask_wtf.file import FileField
+import csv
+import io
+
+
+class PhotoForm(FlaskForm):
+    file = FileField()
```

Comment:
в FileField можно передавать аргументами, как его рендерить в html, например вот так уже выглядит симпатичнее
```suggestion
    file = FileField(render_kw={'class': 'form-control'})
```

Reply:
можешь пояснить запись? render_kw - это не обязательный аргумент? что это дает? это в примерах документации есть? я брал прям из нее

Reply:
без render_kw
```html
<input id="file" name="file" type="file">
```
с ним
```html
<input class="form-control" id="file" name="file" type="file">
```
Вот тут документация про поля в формах, это базовый класс, от которого наследуется в том числе FileField https://wtforms.readthedocs.io/en/3.0.x/fields/#the-field-base-class
Тут вот что-то на по-русски на ту же тему https://docs-python.ru/packages/veb-frejmvork-flask-python/ispolzovanie-modulja-wtforms-prilozhenii-flask/